adit/dicom_web/tests/acceptance/test_wadors.py

```diff
@@ -63,6 +123,69 @@ def test_retrieve_study_metadata(channels_live_server: ChannelsLiveServer):
     )
 
 
+@pytest.mark.acceptance
+@pytest.mark.order("last")
+@pytest.mark.django_db(transaction=True)
+def test_retrieve_study_metadata_with_pseudonym(
+    channels_live_server: ChannelsLiveServer,
+):
+    setup_dimse_orthancs()
+
+    _, group, token = create_user_with_dicom_web_group_and_token()
+    server = DicomServer.objects.get(ae_title="ORTHANC1")
+    grant_access(group, server, source=True)
+    orthanc1_client = create_dicom_web_client(channels_live_server.url, server.ae_title, token)
+
+    metadata = load_sample_dicoms_metadata("1001")
+    study_uid: str = metadata["StudyInstanceUID"].iloc[0]
+    test_pseudonym = "TestPseudonym"
+    additional_params = {
+        "pseudonym": test_pseudonym,
+    }
+
+    results = orthanc1_client.retrieve_study_metadata(
```

Comment:
This is where we can use adit_client instead

Reply:
You can just add them. It should be pretty simple, as all those methods are just wrappers around dicomweb_client